dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -4457,7 +4457,8 @@ private Contentlet internalCheckin(Contentlet contentlet,
         }
 
         if (!isCheckInSafe(contentRelationships)){
-            this.esReadOnlyMonitor.start();
+            Try.of(()->DotConcurrentFactory.getInstance().getSingleSubmitter().submit(()-> this.elasticReadOnlyCommand.executeCheck()))
+                    .onFailure(e -> Logger.error(ESContentletAPIImpl.class, e.getMessage()));
             throw new DotContentletStateException(
                     "Content cannot be saved at this moment. Reason: Elastic Search cluster is in read only mode.");
```

Comment:
As an additional change, can you please add the content ID to the error message?